spec/models/schedule_spec.rb

```diff
@@ -2,17 +2,17 @@
 
 RSpec.describe Schedule, type: :model do
   describe 'validation' do
-    context '#from' do
+    context '#start_at' do
       context 'first schedule' do
         let(:user) do
           create(:user)
         end
 
-        def build_schedule(from)
-          build(:schedule, user: user, from: from)
+        def build_schedule(start_at)
+          build(:schedule, user: user, start_at: start_at)
```

Comment:
General observation: for lots of these you could use `build_stubbed` instead as they don't roundtrip.